testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/ManagementReadXmlTestCase.java

```diff
@@ -52,6 +52,7 @@
  *
  * @author Emmanuel Hugonnet (c) 2022 Red Hat, Inc.
  */
+@org.junit.Ignore("WFCORE-6679")
```

Comment:
@pferraro please rebase as this test is fixed

Reply:
Will do.